src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    add: function(number) {
+      obj.result = obj.result + number;
```

Comment:
```suggestion
      obj.result += number;
```
Можна використовувати такий скорочений запис (з іншими операціями можна так само)